packages/web/src/components/pages/WebsiteResults.tsx

```diff
@@ -125,6 +125,15 @@ export function WebsiteResultsPage() {
     (packageStats.vulnerable > 0 ? `, ${packageStats.vulnerable} are vulnerable` : '') +
     (packageStats.outdated > 0 ? `, ${packageStats.outdated} are outdated` : '');
 
+  const webpackVersion = scanResult?.scan?.scanResult?.identifiedBundler?.versionRange ?? 'x.x';
+  const accuracy = // TODO: remove hardcode
+    {
```

Comment:
nit: Moving the object into module-level scope would improve readability (and perf by a lesser degree)